src/zenml/exceptions.py

```diff
@@ -156,3 +156,14 @@ def __init__(
 
 class IntegrationError(Exception):
     """Raises exceptions when a requested integration can not be activated."""
+
+
+class DuplicateRunNameError(RuntimeError):
+    """Raises exception when a run with the same name already exists."""
```

Comment:
Maybe we can call this a `DuplicateError` instead to generalize to other Duplicate use-cases?

Reply:
Duplicate in general or some kind of Metadata store duplicate? `DuplicateError` sounds a little too generic to me, but I agree that like this it's maybe a little too specific.

Reply:
I was thinking maybe there are other duplicates like `DuplicateStepName` or something. You can make scoped to the Metadata store if you'd like

Reply:
What do you think of `DuplicateContextError`? Then it can sort of refer to anything in the metadata store?

Reply:
Won't that conflict with the notion of a `StepContext`?

Reply:
Mhm yeah, any other suggestions that aren't as broad as `DuplicateError`?

Reply:
Lets leave it like this and merge